chapter_1/1_is_unique/lib/is_unique.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Implement an algorithm to determine if
+# a string has all unique characters. What if you
+# cannot use additional data structures?
+
+class IsUnique
+  def check_unique(string)
```

Comment:
I like this solution. It's definitely the most semantic in Ruby. I'm going to challenge you by asking how would you solve this without `uniq` some languages (like Java) don't have a built in function like that

Reply:
make sense, ruby makes things too easy sometimes, I'll rework it without `uniq`